.travis.yml

```diff
@@ -68,8 +68,8 @@ jobs:
       env: MATRIX_EVAL="CC=clang-7 && CXX=clang++-7" xRUN_BUILD_AND_TESTSUITE=1
     - stage: OsX
       os: osx
-      osx_image: xcode10.2
-      language: generic
+      osx_image: xcode10.3 # Python 3.7.4 running on macOS 10.14.4
+      language: shell # 'language: python' is an error on Travis CI macOS
```

Comment:
```suggestion
      language: shell      # languages 'generic' or 'python' result in errors on macOS
```

Reply:
@jougs Thanks for reviewing.  `language: generic` works in general for OsX. Our current setup uses it.